public_html/about.php

```diff
@@ -62,8 +70,52 @@
     $contributors_html .= '</div></div>';
 }
 $contributors_html .= '</div>';
+$contributors_html .= '<div style="display: none;" id="locations">' . json_encode($locations) . '</div>';
```

Comment:
This is fine, but I'd usually do it as a `script` block instead:

```php
$contributors_html .= '<script type="text/javascript">var locations = '.json_encode($locations).'</script>';
```

Then below you can just access the `locations` variable directly, instead of having to call the `parseJSON` function.